docs/source/conf.py

```diff
@@ -13,10 +13,15 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parent
```

Comment:
Do you need to do this path trick? Can't you just put them under `_static`? https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_favicon says the path should be relative the conf file?

Reply:
I updated the docs to use `_static`.